Training event (#1203)
nf-core/nf-co.re: markdown/events/2022/training-october-2022.md

---
title: nf-core Training - October 2022
subtitle: A virtual Nextflow and nf-core training event
type: workshop
start_date: '2022-10-03'
start_time: '03:00 CEST'
end_date: '2022-10-05'
end_time: '21:30 CEST'

> location_name: Virtual

Reviewer: ```suggestion
location_name: Virtual (Gather.town)
```

> import_typeform: true

Reviewer: Don't think that we need this - pretty sure it was just when we were embedding typeform surveys in the page before.
```suggestion
```

---

# Welcome

> Join us October 3-5 for the nf-core training event!

Reviewer: ```suggestion
Join us from October 3-5, 2022 for the Nextflow / nf-core training event!
```

The training is entirely virtual and free and will cover the fundamentals of using Nextflow and nf-core. The training will offer something for all skill levels but will be especially useful for those who are new to Nextflow and the nf-core community, or if you are thinking about joining the [nf-core Hackathon](https://nf-co.re/events/2022/hackathon-october-2022) (October 10-12, 2022) for the first time.

# Schedule

> The training will be held across three consecutive days in 2.5-hour sessions (including time for a small break and questions). Because nf-core is a global community we are making the training geographically accessible and will be holding separate sessions for different timezones:

Reviewer: ```suggestion
The training will be held across three consecutive days in 2.5-hour sessions (including time for a small break and questions).

Because nf-core is a global community we are making the training geographically accessible and will be holding separate sessions for different timezones:
```

            </tr>
        </thead>
        <tbody>
            <tr>
                <td>:earth_asia: APAC (Asia-Pacific)</td>
                <td data-timestamp="1664758800" data-timeformat="DD-MMM HH:mm z"></td>
                <td data-timestamp="1664845200" data-timeformat="DD-MMM HH:mm z"></td>
                <td data-timestamp="1664931600" data-timeformat="DD-MMM HH:mm z"></td>
            </tr>
            <tr>
                <td>:earth_africa: EMEA (Europe, the Middle East, and Africa)</td>
                <td data-timestamp="1664798400" data-timeformat="DD-MMM HH:mm z"></td>
                <td data-timestamp="1664884800" data-timeformat="DD-MMM HH:mm z"></td>
                <td data-timestamp="1664971200" data-timeformat="DD-MMM HH:mm z"></td>
            </tr>
            <tr>
                <td>:earth_americas: AMER (North, Central, and South America)</td>
                <td data-timestamp="1664816400" data-timeformat="DD-MMM HH:mm z"></td>
                <td data-timestamp="1664902800" data-timeformat="DD-MMM HH:mm z"></td>
                <td data-timestamp="1664989200" data-timeformat="DD-MMM HH:mm z"></td>
            </tr>
        </tbody>
    </table>
</div>

> 

Reviewer: ```suggestion

_(All times shown in your local time zone)_

```

> Of course, if you can’t make the session in your timezone, you are welcome to attend another session at the time that works best for you.

Reviewer: ```suggestion
You are welcome to attend whichever sessions work best for you.
Training will be held online in [Gather.town](https://gather.town) (more details closer to the time).
```
